fastapi_kafka/consumer.py

```diff
@@ -68,14 +67,12 @@ async def wrapper(message, *args, **kwargs):
                     except (json.JSONDecodeError, ValidationError) as e:
                         raise ValueError(f"Kafka message validation error: {e}") from e
                 else:
-                    if self._is_json(message_value):
-                        try:
-                            message_data = json.loads(message_value)
-                            return await func(message_data, *args, **kwargs)
-                        except json.JSONDecodeError as e:
-                            raise ValueError(f"Failed to decode Kafka message: {e}") from e
```

Comment:
This original line seems to be an unreachable piece of code, since if `_is_json(message_value)` evaluates to `True`, then `json.loads(message_value)` cannot even fail.